### ISSUE TITLE
JavadocTagContinuationIndentation doesn't report some lines

### ISSUE THREAD

**Issue by rnveach** [checkstyle/checkstyle#2813]
https://checkstyle.org/config_javadoc.html#JavadocTagContinuationIndentation

TestClass.java

```
package test;

public class TestClass {
    /**
     * Desc.
     * 
     * @param s
     * Desc. Line 8.
     * Desc 2.
     * Desc 3.
     *                         Desc 4. Line 11. violation is expected
     */
    public void test(String s) {
    }
}
```

config.xml

```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="ASCII" />
    <property name="severity" value="error" />
    <property name="fileExtensions" value="java" />

    <module name="TreeWalker">
        <module name="JavadocTagContinuationIndentation" />
    </module>
</module>
```

output from: `java -jar checkstyle-6.14.1-all.jar -c config.xml TestClass.java`

```
Starting audit...
[ERROR] TestClass.java:9: Line continuation have incorrect indentation level, 
    expected level should be 4. [JavadocTagContinuationIndentation]
[ERROR] TestClass.java:10: Line continuation have incorrect indentation level, 
     expected level should be 4. [JavadocTagContinuationIndentation]
Audit done.
Checkstyle ends with 2 errors.
```

Expected **extra** output:

```
[ERROR] TestClass.java:8: Line continuation have incorrect indentation level, 
    expected level should be 4. [JavadocTagContinuationIndentation]
[ERROR] TestClass.java:11: Line continuation have incorrect indentation level,
     expected level should be 4. [JavadocTagContinuationIndentation]
```

I am unsure if line 8 is considered a "continuation" line, since it is technically the first of the description. I, however, still expect it to be indented correctly, as if it was a continuation line, since it is starting on a new line from its `param` tag and it will look nicer to have it at the same indentation as the following lines.
If it not being indented, or placed where ever wanted, is expected and correct behavior, then it should be explained in the xdocs atleast.

Line 11 will be handled in https://github.com/checkstyle/checkstyle/issues/8182 .

**Comment by Gaurav-Punjabi** [comment]
I'm on it.

**Comment by rnveach** [comment]
@romani Please confirm if new functionality should do this part too:
> Line 11 however should be required to be at the same position as previous indented lines, but it seems JavadocTagContinuationIndentation ignores it once it is indented past its minimum, required position.

IndentationCheck has a special option for strict enforcement. Should we move this item to its own issue?
Asked in PR https://github.com/checkstyle/checkstyle/pull/8002#issuecomment-615133226

**Comment by romani** [comment]
this Check, as all other in [version 6.0](https://checkstyle.org/releasenotes.html#Release_6.0) are created for Google style.

https://checkstyle.org/google_style.html#a7.1.3
https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s7.1.3-javadoc-at-clauses

from google style:
> When a block tag doesn't fit on a single line, continuation lines are indented four (or more) spaces from the position of the @. 

main part is `are indented four (or more) spaces` BUT for block tags only.

**Comment by Gaurav-Punjabi** [comment]
> BUT we can create new property forceStrictCondition ( same is in https://checkstyle.org/config_misc.html#Indentation_Properties with default false) and let users to be more strict.

@romani This sounds good 👍

**Comment by rnveach** [comment]
@Gaurav-Punjabi I split the new property into a new issue and removed it from this issue.

**Comment by rnveach** [comment]
Fix was merged
